libs/accounts/src/lib/accounts-table.spec.tsx

```diff
@@ -24,47 +25,104 @@ const singleRow: AccountFieldsFragment = {
     symbol: 'tBTC',
     decimals: 5,
   },
+  available: '125600000',
+  used: '125600000',
+  deposited: '125600000',
 };
 const singleRowData = [singleRow];
 
 describe('AccountsTable', () => {
   it('should render successfully', async () => {
-    await act(async () => {
-      const { baseElement } = render(<AccountsTable data={[]} />);
-      expect(baseElement).toBeTruthy();
-    });
+    const { baseElement } = render(
```

Comment:
chore (blocking): This test is printing out console errors, when testing components that render ag grid they need to be wrapped in act

Suggest something like the following:
```
it('should render successfully', async () => {
    await act(async () => {
      render(<AccountTable rowData={[]} onClickAsset={() => null} />);
    });
    expect(document.querySelector('.ag-root-wrapper')).toBeInTheDocument();
  });
```

Reply:
updated